morphia/src/main/java/org/mongodb/morphia/mapping/Mapper.java

```diff
@@ -737,7 +755,7 @@ private void addConverters(final MappedClass mc) {
     private MappedClass addMappedClass(final MappedClass mc, final boolean validate) {
         addConverters(mc);
 
-        if (validate) {
+        if (validate && !mc.isInterface()) {
```

Comment:
Acknowledged, but still it is orthogonal to collation as far as I know.  So unless I'm wrong about that it should be handled separately.